releases.json

```diff
@@ -985,6 +985,7 @@
       "fmt"
     ],
     "versions": [
+      "11.2.0-2",
```

Comment:
I think you should remove the line below (no need to provide two versions of 11.2.0-wrappers).

Reply:
Nope, existing releases must not be removed.